### PR TITLE
test(infrastructure): Add testing infrastructure

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: test/convertItkImageToVtkImageTest.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  t.end()
+}
+
+test('readImageBlob reads a Blob', t => {
+  return readImageBlob(cthead1SmallBlob, 'cthead1Small.png').then(function (image) {
+    verifyImage(t, image)
+  })
+})
+
+test('convertItkImageToVtkImage results in the expected image', (t) => {
```
Comment (agirault) [inline test/convertItkImageToVtkImageTest.js]: why are there parenthesis around `t` here and not for the test above?

File: test/convertItkImageToVtkImageTest.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  t.ok(true, 'a check passes')
```
Comment (agirault) [inline test/convertItkImageToVtkImageTest.js]: just curious, does that test do anything?

File: test/convertItkImageToVtkImageTest.js (hunk 1 above)
Comment (thewtex) [reply test/convertItkImageToVtkImageTest.js]: The parenthesis are optional, but I will add them to the test above for clarity and consistency.

File: test/convertItkImageToVtkImageTest.js (hunk 2 above)
Comment (thewtex) [reply test/convertItkImageToVtkImageTest.js]: Not yet. "Epic tests coming soon to a repository near you." ;-)

File: test/convertItkImageToVtkImageTest.js (hunk 1 above)
Comment (agirault) [reply test/convertItkImageToVtkImageTest.js]: eslint won't let me 🛌  at night if I don't put these ;)

**Pull request merged**